cobaya/likelihoods/planck_2018_lowl/TT_native.py

```diff
@@ -0,0 +1,157 @@
+import os
+import numpy as np
+from cobaya.likelihoods.base_classes import InstallableLikelihood
+
+class TT_native(InstallableLikelihood):
+    """
+    Python translation of the Planck 2018 Gibbs TT likelihood (python Eirik Gjerløw, Feb 2023)
+    See https://wiki.cosmos.esa.int/planck-legacy-archive/index.php/CMB_spectrum_%26_Likelihood_Code
+
+    """
+
+    install_options = {"github_repository": "CobayaSampler/planck_native_data",
+                       "github_release": "v1",
+                       "asset": "planck_2018_lowT.zip",
+                       "directory": "planck_2018_lowT_native"}
+
+    type = "CMB"
+    aliases = ["lowT"]
+
+    _lmin = 2
+    _lmax = 29 # Could in principle be extended to 200
+    _delta_l = 1000
+    _nbins = 0
+
+
+    data = {}
```

Comment:
Private variables should start with underscore (things that can't be changed via input yaml options) - i.e. data variables like the ones above.

Reply:
Should be fixed now